app/util/data_preparation/confluence/prepare-data.py

```diff
@@ -12,9 +12,9 @@
 USERS = "users"
 PAGES = "pages"
 BLOGS = "blogs"
-DEFAULT_USER_PREFIX = 'performance_'
+DEFAULT_USER_PREFIX = 'performanceAA_'
```

Comment:
I think you forgot to change the prefix back

Reply:
already fix it above ^^^ `fix misstypes` commit